bosh-shell/startup.sh

```diff
@@ -19,7 +19,11 @@ ssh -fNT -4 -L 25555:localhost:25555 \
 bosh -t localhost login admin -- "$BOSH_ADMIN_PASSWORD"
 bosh target localhost
 
-bosh download manifest "${BOSH_DEPLOYMENT}" "${BOSH_DEPLOYMENT}.yml"
-bosh deployment "${BOSH_DEPLOYMENT}.yml"
+if bosh download manifest "${BOSH_DEPLOYMENT}" "${BOSH_DEPLOYMENT}.yml"; then
+  bosh deployment "${BOSH_DEPLOYMENT}.yml"
```

Comment:
Suggested fix if this is an empty file:

```
if bosh download manifest "${BOSH_DEPLOYMENT}" "${BOSH_DEPLOYMENT}.yml"; then
  if [ -s "${BOSH_DEPLOYMENT}.yml" ]; then
    bosh deployment "${BOSH_DEPLOYMENT}.yml"
  else
    echo "Unable to target deployment ${BOSH_DEPLOYMENT}, manifest is an empty file."
  fi
else
  echo "No deployment targetted..."
  echo "You'll need to manually download a manifest and target a deployment."
fi
```

Reply:
Good call. I've added a commit to with this fix.